app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Avatar, Dropdown, Typography } from "@bigbinary/neetoui";
```

Comment:
Create aliases for commonly used paths. https://academy.bigbinary.com/learn-react/react-best-practices/create-alias-for-commonly-used-directories